examples/src/utils/CustomCameraManager.ts

```diff
@@ -20,6 +21,29 @@ export class CustomCameraManager implements CameraManager {
   private readonly _cameraChangedListener: Array<CameraChangeDelegate> = [];
   private _stopEventHandler: DebouncedCameraStopEventTrigger;
 
+  /**
+   * Reusable buffers used by updateNearAndFarPlane function to avoid allocations.
+   */
+  private readonly _updateNearAndFarPlaneBuffers = {
+    cameraPosition: new Vector3(),
+    cameraDirection: new Vector3(),
+    corners: new Array<Vector3>(
+      new Vector3(),
+      new Vector3(),
+      new Vector3(),
+      new Vector3(),
+      new Vector3(),
+      new Vector3(),
+      new Vector3(),
+      new Vector3()
+    )
+  };
+
+  private readonly _calculateCameraFarBuffers = {
+    nearPlaneCoplanarPoint: new Vector3(),
+    nearPlane: new Plane()
+  };
```

Comment:
Can we use a CameraManagerHelper instance here as well, instead of these big field declarations?

Reply:
My bad, fixed.